website/utils/test/url_utils.test.ts

```diff
@@ -413,6 +414,13 @@ describe("Viewer state serialization", () => {
     region: { x: [0, 0.5], y: [0, 1], z: [0, 1] },
     slice: { x: 0.25, y: 0.75, z: 0.5 },
     time: 100,
+    cameraState: {
+      position: [-1.05, -4, 45],
+      target: [0, 0, 0],
+      up: [0, 1, 0],
+      orthoScale: 3.534,
+      fov: 43.5,
+    },
```

Comment:
it seems like for the example of going from the URL to state you should have partial cameraState. ie, position but nothing else

Reply:
We expect all of these to be defined... I think this might need to be a separate test.